docs/embeddable-uis/element/access-request.mdx

```diff
@@ -0,0 +1,421 @@
+---
+sidebar_position: 3
+title: Access Request
+---
+
+import CodeDropdown from "@site/src/components/elements/CodeDropdown.js";
+import code from "@site/src/components/elements/codeBlock.json";
+
+# Access Request Element
+
+<span className="subtitle-custom">
+The Access Request feature allows users to request access to restricted resources. The request is sent to the workspace owner, who can approve or deny it.
+</span>
+<br/>
+<br/>
+
+<span style={{margin:'0 30px'}}>
+<img src="/img/approval-flows/ar-element.png" alt="ar-element" style={{width:'70%',border:'1px solid black'}}/>
+</span>
+
+
+## Overview
+
+Access Request gives us new capability in our app. Now the users can ask to dynamically change their own permissions if they want to access a forbidden action.
+
+**For example**
+
+A normal user can't write a blog article, but an editor can. When a normal user wants to write an article, he requests access to the screen. When the system administrator gets his access request (in the user management element), he decides whether to grant him access or not.
+
+It is necessary to have a User Management Element in place for an access request. This is because when an access request is created, the workspace owner can view it in its user management element. This is where they can manage their list of users and approve access requests.
```

Comment:
This should be under `User Management Element` Section, and before the screenshot